README.md

```diff
@@ -1,50 +1,69 @@
 # BigchainDB Whitepaper
 
+![repo banner](media/repo-banner@2x.png)
+
 This repository contains the source (LaTeX) for the BigchainDB whitepaper.
 
+---
+
+**The most recent PDF version of the whitepaper can always be retrieved from [bigchaindb.com/whitepaper](https://www.bigchaindb.com/whitepaper)**
+
+---
+
 If you're looking for the main BigchainDB repository, it's at [bigchaindb/bigchaindb](https://github.com/bigchaindb/bigchaindb).
 
+And if you're really here to get a glimpse of the exciting world of LaTeX-based file generation, this repo is for you. 
 
-## Prerequisites for Using Bash Script or Make
+## Prerequisites for generating the PDF
 
-1. [TeX Live](https://www.tug.org/texlive/) or similar
-2. pdftk (can be installed using `apt-get install pdftk`)
+Build requires two binaries:
 
+- `pdflatex`
+- `bibtex`
 
-## Using the Bash Script to Generate the PDF
+### Using Docker
 
-```text
-$ cd src
-$ ./renderpaper.sh
+This is the preferred way so you don't have to bother with installing `pdflatex` and `bibtex`, which is especially cumbersome on macOS & Windows:
+
+```bash
+docker-compose up
 ```
 
-## Using Make to Generate the PDF
+This will generate the final PDF and output to `./bigchaindb-whitepaper.pdf`.
 
-There's a very simple [Makefile](src/Makefile) under the `src` directory. To generate the PDF (whitepaper.pdf):
+### Linux
 
-```text
-$ cd src
-$ make all
+```bash
+sudo apt-get install -y texlive texlive-latex-extra pdftk
 ```
 
-The [Makefile](src/Makefile) has a few targets. To clean intermediate files:
+### macOS
+
+On macOS you can get both binaries as part of the BasicTex package, which can be installed manually via [their package installer](http://www.tug.org/mactex/morepackages.html) or via Homebrew cask:
 
-```text
-$ make clean
+```bash
+brew cask install basictex
 ```
 
-## Using Docker to Generate the PDF
+Now you have a whole bunch of binaries in a rather weird location. You can either symlink the required binaries to one of your `PATH`s, e.g. `/usr/local/bin`:
+
+```bash
+# create symlink for pdflatex binary in .app package
+ln -s /Library/TeX/Distributions/.DefaultTeX/Contents/Programs/texbin/pdflatex /usr/local/bin/pdflatex
+# create symlink for bibtex binary in .app package
+ln -s /Library/TeX/Distributions/.DefaultTeX/Contents/Programs/texbin/bibtex /usr/local/bin/bibtex
+```
 
-If you don't wish to bother with installing `pdflatex` and `bibtex`, and feel comfortable with using `docker` and `docker-compose`, then
+Or get all the Tex tools by adding the whole folder to your `PATH` first:
 
-```text
-$ docker-compose up
+```bash
+export PATH=$PATH:/Library/TeX/Distributions/.DefaultTeX/Contents/Programs/texbin
 ```
 
-will generate the PDF.
+## Generate the PDF
```

Comment:
that's a remaining ToDo and if we decide on keeping this as an extra document, the build script should generate that too.

So instead of having one whitepaper we have: whitepaper, addendum & primer. Only whitepaper & addendum are actually being distributed. What's the point of an extra primer document? Couldn't this be integrated into the main whitepaper?